scripts/generate-style-code.js

```diff
@@ -8,8 +8,6 @@ const colorParser = require('csscolorparser');
 
 // FIXME: https://github.com/mapbox/mapbox-gl-native/issues/15008
 delete spec.layout_circle["circle-sort-key"]
-delete spec.layout_line["line-sort-key"]
-delete spec.layout_fill["fill-sort-key"]
```

Comment:
These lines are duplicated in the Darwin version of this script:

https://github.com/mapbox/mapbox-gl-native/blob/1f266b45bc1cf5bec43cbd81a2da587023124401/platform/darwin/scripts/generate-style-code.js#L20-L21

Reply:
mapbox/mapbox-gl-native-ios#179 updates the gl-native-ios repository to reflect this change. @tobrun, heads-up that the Android map SDK is missing runtime styling methods due to similar delete statements in platform/android/scripts/generate-style-code.js.

Reply:
@Chaoba could you follow up on above?